hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/clients/OfflineClientsService.java

```diff
@@ -1,10 +1,5 @@
 package pl.allegro.tech.hermes.management.domain.clients;
 
-import pl.allegro.tech.hermes.api.TopicName;
-
-import java.util.List;
-
 public interface OfflineClientsService {
-
-    List<OfflineClient> find(TopicName topic);
+    String getIframeSource(String topic);
```

Comment:
Why you've changed `TopicName` parameter type to `String`?

Reply:
Thats because i just wanted to avoid unnecessary `TopicName.fromQualifiedName(topic)` in controller, like it was before.

Reply:
I'm not sure if it is unnecessary - this method correctly verifies if topic is valid:
```java
public static TopicName fromQualifiedName(String qualifiedName) {
    if (Strings.isNullOrEmpty(qualifiedName)) {
        return null;
    }

    int index = qualifiedName.lastIndexOf(GROUP_SEPARATOR);
    if (index == -1) {
        throw new IllegalArgumentException("Missing group");
    }

    String groupName = qualifiedName.substring(0, index);
    String topicName = qualifiedName.substring(index + 1, qualifiedName.length());
    return new TopicName(groupName, topicName);
}
```

also, all methods in `TopicsEndpoint` use this validation. I would add creating `TopicName` in `OfflineClientsEndpoint` so it would validate topic correctness on API layer. For example, someone can call this `offline-clients-source` endpoint manually (with invalid topic name).

Reply:
You're right, I fixed it